src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -137,11 +149,11 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundIfStatement : IIfStatement
     {
-        IExpression IIfStatement.Condition => this.Condition;
+        IOperation IIfStatement.Condition => this.Condition;
 
-        IStatement IIfStatement.IfTrue => this.Consequence;
+        IOperation IIfStatement.IfTrue => this.Consequence;
```

Comment:
You had to notice. How about "IfTrueValue" and "IfFalseValue"?

I liked the older names better too.